client/src/modules/Job/pages/NoAuthLanding/components/TopSlide.tsx

```diff
@@ -0,0 +1,134 @@
+import { COLORS } from 'modules/Job/theme/colors';
+import { DottedRectangle } from './DottedRectangle';
+
+export function TopSlide() {
+  return (
+    <>
+      <div className="wrapper">
+        <div className="divider"></div>
+        <div className="content">
+          <div className="text-wrapper">
+            <div className="logo-dots-wrap">
+              <div className="dot-wrap">
+                <DottedRectangle />
+              </div>
+              <img src="/static/svg/jobs/rs-logo-big.svg" className="logo" alt="logo-rss" />
+            </div>
+            <div className="text-block">
+              <h1 className="title"> RS Jobs - developer job portal</h1>
+              <p className="subtitle"> powered by The Rolling Scopes developer community</p>
+            </div>
+          </div>
+          <div className="circle-logo-wrap">
+            <div className="circle"></div>
+            <img src="/static/svg/jobs/sloth-welcome.svg" className="welcome-pic" alt="welcome" />
```

Comment:
we have this image
see the usage example here
https://github.com/rolling-scopes/rsschool-app/blob/master/client/src/modules/TeamDistribution/components/WelcomeCard/WelcomeCard.tsx
string 32

Reply:
swapped it